### PR TITLE
Finnish translations

### PR DESCRIPTION
**Pull request by tatupesonen**

### CODE REVIEW

File: fi.json [hunk 1]
```diff
@@ -82,301 +82,301 @@
...
-            "option-frame-description": "Determines the styling of the title bar of Obsidian windows. Requires a full restart to take effect.",
-            "option-frame-hidden": "Hidden (default)",
-            "option-frame-obsidian": "Obsidian frame",
-            "option-frame-native": "Native frame",
-            "option-configure-ribbon": "Ribbon menu",
-            "option-configure-ribbon-desc": "Configure what commands appear in the ribbon menu in that navigation bar.",
+            "name": "Ulkonäkö",
+            "option-base-theme": "Pohjateema",
+            "option-base-theme-description": "Valitse Obsidianin oletusväriteema.",
+            "option-accent-color": "Aksenttiväri",
```
Comment (evanyholm) [inline fi.json:130]: Korostusväri

File: fi.json [hunk 2]
```diff
@@ -82,301 +82,301 @@
...
-            "tooltip-customize-command": "Customize this command",
-            "tooltip-hotkey-single-conflict": "This hotkey conflicts with \"{{command}}\"",
-            "tooltip-hotkey-multiple-conflicts": "This hotkey conflicts with {{count}} other commands",
-            "label-blank-hotkey": "Blank"
+            "name": "Pikanäppäimet",
+            "prompt-filter": "Suodata...",
+            "tooltip-delete-hotkey": "Poista pikanäppäin",
+            "label-waiting-for-hotkey-press": "Paina pikanäppäintä...",
+            "tooltip-restore-default": "Palauta oletus",
+            "tooltip-customize-command": "Kustomoi tätä komentoa",
```
Comment (evanyholm) [inline fi.json:210]: Olisiko "muokkaa" parempi ja vähemmän finglish?

File: fi.json (hunk 1 above)
Comment (tatupesonen) [reply fi.json:130]: Hello, thanks for the suggestion! Feel free to submit a PR to the source repo :)

**Comment by tatupesonen** [comment]
Here's some initial translations, ready to be merged. By no means complete but it's a start.

**Comment by joethei** [comment]
Thank you for your work

**Pull request merged**